### PR TITLE
[DOC] Fix broken links

### PR DESCRIPTION
**Pull request by BurdetteLamar**

### CODE REVIEW

File: lib/csv.rb [hunk 1]
```diff
@@ -106,7 +106,7 @@
 #
 # If you are familiar with \CSV data and have a particular task in mind,
 # you may want to go directly to the:
-# - {Recipes for CSV}[doc/csv/recipes/recipes_rdoc.html].
+# - {Recipes for CSV}[https://doc.rubylang.org/doc/csv/recipes/recipes_rdoc.html].
```
Comment (kou) [inline lib/csv.rb:109]: Hmm. If we do this, it doesn't refer a file in the same version...

Comment (BurdetteLamar) [reply lib/csv.rb:109]: Sorry, I meant to remove this change.  I've submitted https://github.com/ruby/csv/issues/284 for this link.

Comment (BurdetteLamar) [reply lib/csv.rb]: This link is now removed in this PR.  The recipes are not available in ruby/ruby.

Comment (kou) [reply lib/csv.rb]: OK.

**Pull request merged**